packages/generate/src/cli.ts

```diff
@@ -367,29 +367,33 @@ Run this command inside an EdgeDB project directory or specify the desired targe
     exitWithError(`Failed to connect: ${(e as Error).message}`);
   }
 
-  switch (generator) {
-    case Generator.QueryBuilder:
-      await generateQueryBuilder({
-        options,
-        client,
-        root: projectRoot,
-      });
-      adapter.process.exit();
-      break;
-    case Generator.Queries:
-      await generateQueryFiles({
-        options,
-        client,
-        root: projectRoot,
-      });
-      break;
-    case Generator.Interfaces:
-      await runInterfacesGenerator({
-        options,
-        client,
-        root: projectRoot,
-      });
-      break;
+  try {
+    switch (generator) {
+      case Generator.QueryBuilder:
+        await generateQueryBuilder({
+          options,
+          client,
+          root: projectRoot,
+        });
+        adapter.process.exit();
```

Comment:
I think we should explicitly exit from the process in the CLI for all generators.

Can we move this to a `finally` block?

Reply:
Ah I missed that one. I'll just drop it down to after the switch block. The finally isn't needed as the catch exits.